src/blocks/utils/SeamMeme/assets/icons.ts

```diff
@@ -0,0 +1,33 @@
+import UndoIcon from '@mui/icons-material/Undo';
```

Comment:
No need to include this as a separate file + typed constants -- it adds extra complexity. Rather, just include the imports in the files you need them.